CMakeLists.txt

```diff
@@ -146,6 +146,7 @@ if(MSVC)
   add_compile_definitions(_SILENCE_CXX17_OLD_ALLOCATOR_MEMBERS_DEPRECATION_WARNING)
   add_compile_definitions(_SILENCE_CXX17_RESULT_OF_DEPRECATION_WARNING)
 
+  add_compile_options(/Ot)     # Optimize for speed
```

Comment:
Is this for debug builds? /Ot is already implied for optimized builds

Reply:
A leftover from my tests.
Will remove